data_sources/Production Youth Permissions.csv

```diff
@@ -84,6 +84,8 @@
 02238,Cambridge,smintz@cambridgema.gov,gkotowski@cambridgema.gov,jking@cambridgema.gov,,,,,,,,,,,,,tpass@cambridgema.gov
 02150,Chelsea,rutha@greenrootschelsea.org,camiloa@greenrootschelsea.org,,,,,,,,,,,,,,youthpass@greenrootschelsea.org
 02149,Everett,andrea.romboli@ci.everett.ma.us,,,,,,,,,,,,,,,andrea.romboli@ci.everett.ma.us

```

Comment:
Suggestion: We want to keep this CSV organized based on zip code, not contacts. That means we'll need to update line 86 and then remove lines 87-88

```suggestion
02149,Everett,andrea.romboli@ci.everett.ma.us,catherine.connor@ci.everett.ma.us,joanne.lamonica@ci.everett.ma.us,,,,,,,,,,,,,andrea.romboli@ci.everett.ma.us

```

Reply:
word, fixed here: https://github.com/mbta/reduced_fares/pull/357/commits/ed53461e5d486b3b4142c6b1c97427eb9eb14ca2